site/learn/Learn-Execution.md

```diff
@@ -70,7 +74,7 @@ Query: {
 
 This example is written in JavaScript, however GraphQL servers can be built in [many different languages](/code/). A resolver function receives three arguments:
 
-- `obj` The previous object, which for a field on the root Query type is often not used.
+- `obj` The previous object (also called `RootValue`), which for a field on the root Query type is often not used.
```

Comment:
it is only the root value in the case where this resolver is on the root type. e.g. a root value is provided to correspond to the Query type, but a value for Human type in this page's example is not accurately called a RootValue